ts/features/itwallet/presentation/remote/screens/ItwRemoteClaimsDisclosureScreen.tsx

```diff
@@ -92,25 +96,31 @@ const ContentView = () => {
           />
         </View>
       </View>
-      <ItwOptionalClaimsList
-        items={ItwRemotePresentationClaimsMock.optional}
-        selectedClaims={selectedOptionalClaims}
-        onSelectionChange={toggleOptionalClaims}
+      <ClaimsSelector
+        title="Patente di guida"
+        items={mapMockClaims(ItwRemotePresentationClaimsMock.optional)}
+        selectedItemIds={selectedOptionalClaims}
+        onItemSelected={toggleOptionalClaims}
+        defaultExpanded
+      />
+      <ClaimsSelector
+        title="Credenziale mancante"
+        items={mapMockClaims(ItwRemotePresentationClaimsMock.optional, true)}
+        selectionEnabled={false}
+        defaultExpanded
+      />
+      <Alert
+        variant="info"
+        content={I18n.t(
+          "features.itWallet.presentation.selectiveDisclosure.optionalClaimsAlert"
+        )}
+      />
```

Comment:
As per design, we should hide this alert if all optional claims are selected. Am I right?

Reply:
It seems so, but I think there are discrepancies in the design: see [here](https://www.figma.com/design/pVD9XCba7crkg5yOIdJCXl/branch/42fUgDf4iduVGBNZ9rhsK7/%5BIT-Wallet%5D---%E2%9C%85-Approved-only-%E2%9C%85?node-id=13708-6869&t=sZEyYOK7t7NOd2pu-0).

However there is a bit of uncertainty about optional claims if we consider the DQCL specs, that do not provide an explicit way to mark claims as optional. I think we're inevitably going to rework this screen in the future, so I'd leave it like this if you agree.